api/propagation/noop_propagator.go

```diff
@@ -0,0 +1,34 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package propagation
+
+import (
+	"context"
+)
+
+type NoopPropagator struct {
```

Comment:
I think we can get away with not exporting this propagator. I think the only time this will be used is if nothing is registered with the global package. I don't see a case where a user will need to directly use this propagator. I order to reduce the overall API surface and minimize the future maintenance can we move this to the internal package of the global package? That way it is unexported by the behavior is provided by default.

Reply:
Hi, @MrAlias I moved the file to `api/global/internal`. Does this address your comment or I miss anything? Thanks.